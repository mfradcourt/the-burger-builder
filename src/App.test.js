import React from 'react';
import { shallow } from "enzyme";

import App from './App';

describe('<App />', () => {
  let wrapper;

  it('should render a default App', () => {
    wrapper = shallow(
      <App/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
