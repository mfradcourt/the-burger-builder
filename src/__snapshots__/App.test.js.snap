// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render a default App 1`] = `
<div>
  <layout>
    <Switch>
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.memo),
            "WrappedComponent": [Function],
            "compare": null,
            "displayName": "Connect(Checkout)",
            "type": [Function],
          }
        }
        path="/checkout"
      />
      <Route
        component={[Function]}
        path="/orders"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.memo),
            "WrappedComponent": [Function],
            "compare": null,
            "displayName": "Connect(_temp)",
            "type": [Function],
          }
        }
        exact={true}
        path="/"
      />
    </Switch>
  </layout>
</div>
`;
